### PR TITLE
Add materializing subset to asset gql

### PR DESCRIPTION
**Pull request by johannkm**
Add `materializingPartitions` and `numMaterializing` to the existing resolvers

When we flatten, materializing has the highest priority, then failed, then materialized

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #13045** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13045" target="_blank">Graphite</a> 
    * **PR #13046** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/13046" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/13046?utm_source=stack-comment).

**Comment by johannkm** [comment]
I don't understand the giant snapshot change. It seems like they got reordered? Why does that happen when you just add a new asset and job?

**Review by bengotow — APPROVED** [review]
This looks great! I'll start work on the UI for these changes tomorrow 🙌

File: python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py [hunk 1]
```diff
@@ -461,6 +436,9 @@ def build_partition_statuses(
                 PartitionRangeStatus.FAILED: cast(
                     TimeWindowPartitionsSubset, failed_partitions_subset
                 ),
+                PartitionRangeStatus.MATERIALIZING: cast(
```
[the remark this answers is not in the document]
Comment (johannkm) [reply python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py:439]: added last week 🙂

**Pull request merged**